pkg/telemetry/telemetry.go

```diff
@@ -66,6 +80,10 @@ func NewEngine(conn pgxconn.PgxConn, uuid [16]byte) (*engineImpl, error) {
 	return t, nil
 }
 
+func durationMilliseconds(d time.Duration) int64 {
```

Comment:
is this not already defined somewhere?

Reply:
https://pkg.go.dev/time#Duration.Milliseconds

Reply:
Yes.

> https://pkg.go.dev/time#Duration.Milliseconds

This does not return the converted millisecond of `d`, rather the millisecond'th of `d`